test/files/pos/t11726/SD_1.java

```diff
@@ -0,0 +1,8 @@
+package jli;
+
+public class SD_1 extends CS_1<String, SD_1>.R {
+    public static class Q extends CS_1<String, SD_1> {}
+    public SD_1() {
+        new Q().super();
+    }
+}
```

Comment:
Weird lack of EOL.

Reply:
Really, it's a weird EOL in the other files.